scipy/stats/tests/test_mstats_basic.py

```diff
@@ -355,6 +355,30 @@ def test_kendalltau_seasonal(self):
         assert_almost_equal(output['seasonal p-value'].round(2),
                             [0.18,0.53,0.20,0.04])
 
+
+def test_kendall_tau_large():
+    # Test for the exact method with large samples (n > 171)
+    # expected values generated using SymPy
+    expectations = {100: (-0.033131313131313136, 0.62822615287956040664),
+                    101: (0.03524752475247526, 0.60439525773513602669),
+                    200: (-0.015376884422110552, 0.74753983745929675209),
```

Comment:
I might have expected some tests at the cutover, so n=170 and 171.

Reply:
The algorithm changes at n=171.  That makes it good location to catch regressions.
Is there any test that can be done at those 2 values that would show that they exhibit some correct monotonicity behaviour? E.g. Are there pairs (170, c1) and (171, c2) where you expect the `_kendall_p_exact(...)` values to be "really close" if working correctly?

Reply:
To get the same p-value, we would have to find two samples s1 and s2 (of size 170 and 171, respectively), such that `171*sum(new)_s1 = sum(new)_s2`. I'm not sure how to generate such samples (or the corresponding values of `c`), but I try to think of a method.